src/NServiceBus.Transport.Msmq/MessagePump.cs

```diff
@@ -85,6 +85,28 @@ public Task StartReceive(CancellationToken cancellationToken = default)
             return Task.CompletedTask;
         }
 
+        public async Task ChangeConcurrency(PushRuntimeSettings newLimitations, CancellationToken cancellationToken = new CancellationToken())
```

Comment:
Why ` = new CancellationToken()`? Shouldn't this be `default`?

Reply:
👍